### PR TITLE
Feature/log55 login authentication

### PR DESCRIPTION
**Pull request by Joshua-Burdick**
Made slight UI changes (ion-toolbar -> h3) to make a cleaner header to the sign-in page.

Sign-in page:
- Replaced "Sign in with" buttons
- Added AWS Amplify

Amplify provides user creation/management, such as authentication. Users are prompted to enter a phone number (since it will be on mobile platforms), and an email which is used for verification (SMS verification may come in the future; requires further applications within AWS to be approved for/set up the services).

Added a temporary button to the home screen which will redirect to the Sign-In page (otherwise it cannot be accessed on mobile platforms). This will be phased out and placed as the first redirect users experience, upon sign in they will be redirected to the home screen. This will be coming once the rest of the introductory portions/sign in is created.

### CODE REVIEW

**Comment by Yonava** [comment]
Screen Shot 2023-01-25 at 1 15 50 PM

File: src/views/SignInTab.vue [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    setup() {
+        return {
+        }
+    },
+    mounted() {
+        // Listen for Auth events
+        const AuthListener = (data: any) => {
+            switch (data.payload.event) {
+                case 'signIn':
+                    console.log('User signed in.');
```
Comment (Raidienaty) [inline src/views/SignInTab.vue:57]: Debug print?

File: src/views/SignInTab.vue [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    },
+    methods: {
+        async signOut() {
+            await Auth.signOut();
+        },
+        async deleteUser() {
+            try {
+                const user = await Auth.currentAuthenticatedUser();
+                await Auth.deleteUser();
+                console.log(`User ${user.username} deleted.`);
```
Comment (Raidienaty) [inline src/views/SignInTab.vue:75]: Debug print?

**Comment by Joshua-Burdick** [comment]
> Screen Shot 2023-01-25 at 1 15 50 PM

This issue is fixed

File: src/views/SignInTab.vue [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+});
+
+export default defineComponent({
+    components: {
+        IonPage,
+        IonContent,
+        IonHeader,
+        Authenticator
+    },
+    setup() {
```
Comment (Raidienaty) [inline src/views/SignInTab.vue:48]: Unused method?

**Review by Raidienaty — COMMENTED** [review]
Let some comments.

File: src/views/SignInTab.vue (hunk 1 above)
Comment (Joshua-Burdick) [reply src/views/SignInTab.vue]: I left those there originally to be a clear place to attach the users API to backend. They are now just comments

Comment (Joshua-Burdick) [reply src/views/SignInTab.vue]: I left those there originally to be a clear place to attach the users API to backend. They are now just comments

**Review by Raidienaty — APPROVED** [review]
Comments resolved, marking as approved.

**Pull request merged**